backend/src/main/java/com/example/backend/models/Product.java

```diff
@@ -0,0 +1,61 @@
+package com.example.backend.models;
+
+import java.util.Date;
+
+public class Product {
+    private long id;
+    private String name;
+    private float price;
```

Comment:
float is not the best option for "amount" fields, precision can be a problem. I recommend to use BigDecimal instead